docs/root/version_history/v1.16.1.rst

```diff
@@ -0,0 +1,31 @@
+1.16.1 (November 20, 2020)
+==========================
+
+Incompatible Behavior Changes
```

Comment:
remove unused sections here and below?

Reply:
I agree, that it produces empty sections, but it serves perfectly as a template. When a new release starts, it is enough to copy previous version, delete few lines and the file is ready to receive new lists of Bug Fixes, Deprecated, Minor Behavior Changes, etc. If we remove it, it may lead to inconsistencies. WDYT?

Reply:
Hm, I had assumed we'd been using the template as described in the release process
https://github.com/envoyproxy/envoy/blob/master/GOVERNANCE.md#cutting-a-release
as we do when cutting a new release.

Looking at previous point releases I'm not seeing the empty sections so I'd be inclined to remove the blank lines, and put a pointer to the GOVERNANCE template in the release instructions.  Or if it's easier I can check it in as a template somewhere, and we can alter both GOVERNANCE instructions and stable release instructions to "copy this file and..."

WDYT?  If you still think this is easier I'm fine if you want to tag another maintainer as tie breaker.

Reply:
I will remove empty sections. 

> Or if it's easier I can check it in as a template somewhere

Yeah, I was thinking about the same. It would be best to have a template file.

Reply:
Removed empty sections in releases 1.16.1 and 1.16.2 and updated GOVERNANCE.md file with instructions to delete empty sections when cutting a new release.